documentation/src/main/asciidoc/topics/multimapcache.adoc

```diff
@@ -14,14 +14,15 @@ include::dependencies_maven/multimap.xml[]
 == MultimapCache API
 
 MultimapCache API exposes several methods to interact with the Multimap Cache.
-All these methods are non-blocking in most of the cases. See [limitations]
+These methods are non-blocking in most cases; see
+link:#multimap_limitations[limitations] for more information.
```

Comment:
the link isn't working.

Reply:
argh. it'd help if I added the anchor. fixed now. sorry.